samples/multiplatform/build.gradle.kts

```diff
@@ -0,0 +1,67 @@
+import org.gradle.kotlin.dsl.kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    base
```

Comment:
why using base? 🤔

Reply:
Without it "archives" task do not work

Reply:
[readme.txt](https://github.com/gradle/kotlin-dsl-samples/files/4065110/readme.txt)